src/middlewares/versioning.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+// This middleware ensures that the request specifies the versioning header and
+// its value is the expected value.
+// In the future we'll want to implement an upgrade mechanism.
+import type { Context } from 'koa';
+
+export const ACCEPT_VALUE_PREFIX =
```

Comment:
Please document this header value. It would be good to add some inline comments with the parts. I did a bit of research into why this was chosen, and it seems to be a `mime` type that specifies a specific version. It would be good to be a bit more explicit about why this approach was chosen.